src/components/Timestamp.tsx

```diff
@@ -0,0 +1,17 @@
+export function Timestamp({ timestamp }: { timestamp: number }) {
+  const formatTime = (timestamp: number): string => {
+    return new Date(timestamp).toLocaleTimeString('en-US', {
+      year: 'numeric',
+      month: 'short',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: '2-digit',
+      timeZoneName: 'short',
+    });
+  };
+  return (
+    <p className="margin-0 margin-bottom-4 font-sans-2xs">
```

Comment:
What a useful component! What do you think about moving the styling out of the component and into the parent view so that this can potentially be reused elsewhere?

Reply:
good idea!